metamorph/library/message_data_extractor.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/python
+ANSIBLE_METADATA = {'status': ['preview'],
+                    'supported_by': 'community',
+                    'version': '0.1'}
+
+DOCUMENTATION = '''
+---
+module: message_data_extractor
+short_description: Get important metadata from CI-message.
+version_added: 2.3.0
+author: Jiri Kulda (@Jurisak)
+options:
+  ci-message:
+    description:
+      - Input CI message in json format.
+    required: true
+
+  output:
+    description:
+      - Output metadata file name where CI Message metadata will be stored.
+    required: false
+    default: metamorph.json
+'''
+
+EXAMPLES = '''
+- name: Get metadata from CI-message
+  message_data_extractor:
+    ci-message: path_to/ci-message.json
+  register: result
+'''
+
+RETURN = '''
+Error message:
+  description: An error occurred during metadata extraction
+  returned: Error
+  type: dictionary
+  sample:  { "changed": false, "failed": true, "module_stderr": Given CI_message does not contain
+                                                                important key values. Missing key value: 'header'"
+CI-message metadata:
+  description: Extracted CI-message metadata
+  returned: changed
+  type: dictionary
+  sample: { "changed": true, "meta": { "messages": { "owner": "jkulda",
+                                                     "package": "setup",
+                                                     "release": "5.el7_1",
+                                                     "scratch": "true",
+                                                     "target": "rhel-*-*",
+                                                     "version": "2.8.71"}}
+'''
+
+from metamorph.lib.support_functions import setup_logging, read_json_file, storing_pretty_json
+import logging
+import json
+from ansible.module_utils.basic import AnsibleModule
```

Comment:
suggestion: separate std library, 3rd party, and project local imports with a blank line.